tests/manage/pv_services/test_pvc_disruptive.py

```diff
@@ -157,29 +147,67 @@ def verify_resource_creation(self, func_to_use, previous_num, namespace):
         except TimeoutExpiredError:
             return False
 
+    def pods_creation(self, pvc_objs, pod_factory, interface):
+        """
+        Create pods
+
+        Args:
+            pvc_objs (list): List of ocs_ci.ocs.resources.pvc.PVC instances
+            pvc_objs (function): Function to be used for creating pods
+            interface (int): Interface type
+
+        Returns:
+            list: list of Pod objects
+        """
+        pod_objs = []
+
+        # Create one pod using each RWO PVC and two pods using each RWX PVC
+        for pvc_obj in pvc_objs:
+            if pvc_obj.access_mode == constants.ACCESS_MODE_RWX:
+                pod_obj = pod_factory(
+                    interface=interface, pvc=pvc_obj, status=""
+                )
+                pod_objs.append(pod_obj)
+            pod_obj = pod_factory(interface=interface, pvc=pvc_obj, status="")
+            pod_objs.append(pod_obj)
+
+        return pod_objs
+
     def test_pvc_disruptive(
-        self, storageclass, namespace, interface,
-        operation_to_disrupt, resource_to_delete, teardown_factory
+        self, interface, operation_to_disrupt, resource_to_delete,
+        setup, multi_pvc_factory, pod_factory
```

Comment:
instead of injecting setup to the test function as an argument, you can mark the 'setup()' fixture as autouse=True.
See an example here - https://github.com/red-hat-storage/ocs-ci/pull/679/files#diff-8a9caed0105512cc933374700d306670R24

Reply:
Modified accordingly